packages/loot-core/src/server/accounts/transaction-rules.js

```diff
@@ -90,6 +90,11 @@ export const ruleModel = {
         throw new Error('Invalid rule stage: ' + rule.stage);
       }
     }
+    if (!update || 'conditionsOp' in rule) {
+      if (rule.conditionsOp !== 'and' && rule.conditionsOp !== 'or') {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
      if (!['and', 'or'].includes(rule.conditionsOp)) {
```